src/resources/packages/classy/functions/classy.ts

```diff
@@ -108,19 +107,6 @@ export function toggleElementVisibility( document: Document | null = null ): voi
 export async function initApp( document: Document | null = null ): Promise< void > {
 	document = document ?? window.document;
 	const classyRoot = createRoot( getOrCreateElement( document ) );
-	const registry: WPDataRegistry = await createClassyRegistry();
-
-	// Set the module registry instance.
-	setRegistry( registry );
-
+	const registry = await getDefaultRegistry();
```

Comment:
Should this be typed as `WPDataRegistry`, or is that undesirable since we're getting a Promise back?

Reply:
It's already typed from the promise return value.
Typing it explicitly will not hurt, though.